lib/tests/sessionExists.test.js

```diff
@@ -29,7 +29,7 @@ describe("Test a session middleware", () => {
         email: "test@cds.ca",
         name: "Testing session middleware",
         image: "null",
-        admin: false,
+        role: "program_administrator",
```

Comment:
I noticed in the other test files we import `UserRole` and associate it the `role` property on mockSession.  For commonality we should probalby do that here as well.